stellar/ttl.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const { Contract, SorobanRpc, SorobanDataBuilder, TransactionBuilder, Operation } = require('@stellar/stellar-sdk');
+const { Server } = require('@stellar/stellar-sdk/rpc');
+const { Command } = require('commander');
+const { execSync } = require('child_process');
+const { loadConfig, printInfo, saveConfig } = require('../evm/utils');
+const { stellarCmd, getNetworkPassphrase, getWallet, addBaseOptions } = require('./utils');
+const { getChainConfig, addOptionsToCommands } = require('../common');
+const { prompt } = require('../common/utils');
+require('./cli-utils');
+
+const MAX_INSTANCE_TTL_EXTENSION = 535679;
+
+function checkContractName(chain, contractName) {
```

Comment:
I believe this is being checked in mainProcessor function, no need to double check.